efficientdet/keras/train_lib.py

```diff
@@ -724,7 +751,7 @@ def test_step(self, data):
                                       loss_vals)
       total_loss += det_loss
     if 'segmentation' in self.config.heads:
-      seg_loss_layer = self.loss['seg_loss']
+      seg_loss_layer = self.loss[tf.keras.losses.SparseCategoricalCrossentropy.__name__]
```

Comment:
What's the benefit of using "tf.keras.losses.SparseCategoricalCrossentropy.__name__" instead of a simple "seg_loss"?

Reply:
1. Replace the magic value
2. I could tape into definition when I browse the code.